Screenbox/Converters/ThicknessFiltersConverter.cs

```diff
@@ -142,11 +154,10 @@ public object Convert(object value, Type targetType, object parameter, string la
     {
         if (value is Thickness thickness)
         {
-            var filtersType = Filters;
-            return Convert(thickness, filtersType);
+            return Extract(thickness, Filters);
         }
 
-        return null;
+        return DependencyProperty.UnsetValue;
```

Comment:
I'm not aware of the `DependencyProperty.UnsetValue`. Is there a functional difference between the two?

Reply:
Same here, I only learned about it from this toolkit [converter](https://github.com/CommunityToolkit/Windows/blob/559a2a846a7f1dcd0415a6f3801e85213cb5fa09/components/Converters/src/ColorToDisplayNameConverter.cs#L34). Not sure about the difference, we could just add `#nullable enable` to the file and call it a day. I'm not sure which one is the "correct".